addons/survey/static/src/js/survey_form.js

```diff
@@ -1057,15 +1057,16 @@ publicWidget.registry.SurveyFormWidget = publicWidget.Widget.extend(SurveyPreloa
 
    /**
     * Will automatically focus on the first input to allow the user to complete directly the survey,
-    * without having to manually get the focus (only if the input has the right type - can write something inside -)
+    * without having to manually get the focus (only if the input has the right type - can write something inside -
+    * and if the device is not a mobile device to avoid missing information when the soft keyboard is opened)
     */
     _focusOnFirstInput: function () {
         var $firstTextInput = this.$('.js_question-wrapper').first()  // Take first question
                               .find("input[type='text'],input[type='number'],textarea")  // get 'text' inputs
                               .filter('.form-control')  // needed for the auto-resize
                               .not('.o_survey_comment');  // remove inputs for comments that does not count as answers
-        if ($firstTextInput.length > 0) {
-            $firstTextInput.focus();
+        if ($firstTextInput.length > 0 && !config.device.isMobile) {
+            $firstTextInput.focus();  // do not auto focus on mobile devices
```

Comment:
Don't think it's necessary to have the comment, the condition should be clear enough.